rasa_sdk/interfaces.py

```diff
@@ -384,3 +384,14 @@ def __init__(self, action_name: Text, message: Optional[Text] = None) -> None:
 
     def __str__(self) -> Text:
         return self.message
+
+
+class ActionMissingDomainException(Exception):
+    """Raising this exception when the domain is missing."""
+
+    def __init__(self, action_name: Text, message: Optional[Text] = None) -> None:
+        self.action_name = action_name
+        self.message = message or f"Domain context is missing."
```

Comment:
No need for formatted string. You can ditch the `f` at the start of string.

Reply:
Definitely. CI caught it as well.